### PR TITLE
create contact

### PR DESCRIPTION
**Pull request by Rubens94**
- create contact

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ app.use("/testimonials", testimonialsRouter);
 app.use("/categories", categoriesRouter);
 app.use("/news", newsRouter);
 app.use("/activities", activitiesRouter);
+app.use("/", contactsRouter);
```
Comment (soyarielruiz) [inline app.js]: Index is the same as **contactsRouter**.

File: controllers/contacts.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    async create( req, res) {
+        const { name, phone, email, message } = req.body;
+        await Contact.create({
+            name,
+            phone,
+            email,
+            message
+        });
+
+        try {
```
Comment (soyarielruiz) [inline controllers/contacts.js]: El try deberia ser incluyendo el **Contact.create()**

File: app.js (hunk 1 above)
Comment (Rubens94) [reply app.js]: fixed

File: controllers/contacts.js (hunk 2 above)
Comment (Rubens94) [reply controllers/contacts.js]: fixed

**Review by soyarielruiz — APPROVED** [review]
lgtm

**Pull request merged**